android-automation/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.8
+
+RUN apk add --update
+
+RUN apk add --no-cache python3 \ 
```

Comment:
If you use `python:3.8-alpine`, and merge both next layers, as I suggested before, everything until the `LABEL` will be in a single `RUN` instruction, like

```dockerfile
RUN apk add --no-cache curl && \
    pip install --no-cache-dir click requests
```

Reply:
marged into one layer